pages/common/biff.md

```diff
@@ -0,0 +1,36 @@
+# biff
+
+> A simple utility for doing datetime arithmetic, parsing and formatting.
+> More information: <https://github.com/burntsushi/biff>.
+
+- Print the current time in a format of your choosing:
+
+`biff time fmt {{[-f|--format]}} rfc3339 now`
+
+- Print multiple relative times in one command:
+
+`biff time fmt {{[-f|--format]}} '%c' now -1d 'next sat' 'last monday' '9pm last mon'`
+
+- Print the current time in another time zone, and round it the nearest 15 minute increment:
+
+`biff time in Asia/Bangkok now | biff time round {{[-i|--increment]}} 15 {{[-s|--smallest]}} minute`
+
+- Convert a time between two different time zone:
+
+`TZ='Japan' biff time in America/New_York 02:30`
+
+- Add a duration to the current time:
+
+`biff time add -1w now`
```

Comment:
```suggestion
`biff time add {{-1d|1d|1w|-1m|1y|...}} now`
```

Reply:
is this fine as `{{-1d|1d|1w|-1m|1y|...}}` or should it be `{{[-1d|1d|1w|-1m|1y|...]}}`?

Reply:
{{[ | ]}} is option placeholder syntax. It lets the client determine if it wants to show the long or short options. This has nothing to do with long ans short options.